### PR TITLE
CA-150025: EN: Not provide hotkeys for Tools bar on Workload Reports.

### PR DESCRIPTION
**Pull request by fillzero**
Signed-off-by: Liang Dai <liang.dai1@citrix.com>

### CODE REVIEW

**Comment by fillzero** [comment]
Hi, 
I uploaded screenshots:
https://issues.citrite.net/browse/CA-150025

File: XenAdmin/Wlb/WlbReports/WorkloadReports.resx [hunk 1]
```diff
@@ -583,7 +583,7 @@
     <value>96, 96</value>
   </data>
   <data name="$this.ClientSize" type="System.Drawing.Size, System.Drawing">
-    <value>920, 524</value>
+    <value>950, 524</value>
```
Comment (fillzero) [inline XenAdmin/Wlb/WlbReports/WorkloadReports.resx]: Set the  component's size wider to see more characters

**Comment by cheng-z** [comment]
Hi @fillzero 
I think in XenCenter we handle the hotkey with different way.
See: https://github.com/fillzero/xenadmin/blob/96c565248d5a41b43b74a0e52c4f196956eb3a16/XenModel/Messages.ja.resx#L1182

**Comment by fillzero** [comment]
Close this one for not goog

**Comment by fillzero** [comment]
Close this for not really fix this issue

**Pull request closed without merging**